drivers/scheduler/k8s/specs/kubevirt-simple-vm/kubevirt-app.yaml

```diff
@@ -0,0 +1,39 @@
+apiVersion: kubevirt.io/v1
+kind: VirtualMachine
+metadata:
+  name: testvm
+spec:
+  running: true
+  template:
+    metadata:
+      labels:
+        kubevirt.io/size: small
+        kubevirt.io/domain: testvm
+      annotations:
+        kubevirt.io/keep-launcher-alive-after-failure: "true"
+    spec:
+      domain:
+        devices:
+          disks:
+            - name: containerdisk
+              disk:
+                bus: virtio
+            - name: cloudinitdisk
+              disk:
+                bus: virtio
+          interfaces:
+            - name: default
+              masquerade: {}
+        resources:
+          requests:
+            memory: 64M
```

Comment:
Is 64M enough for the VMs ?

Reply:
This is just a minimal VM with no PVC attached to it and it uses memory as a disk, it is meant for quick testing which ensures that the kubevirt cluster is in a position to provision VMs. Not for real system tests.